app/src/test/java/com/safetyheads/akademiaandroida/PhoneNumberValidatorTest.kt

```diff
@@ -0,0 +1,79 @@
+package com.safetyheads.akademiaandroida
+
+import android.content.Context
+import android.widget.EditText
+import com.safetyheads.akademiaandroida.utils.PhoneNumberValidator
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.assertj.core.api.AssertionsForInterfaceTypes.assertThat
+import org.junit.Test
+
+class PhoneNumberValidatorTest {
+
+    @Test
+    fun `attach should add text changed listener to EditText`() {
+        // Given
+        val editText = mockk<EditText>(relaxed = true)
+        val context = mockk<Context>()
+
+        // When
+        PhoneNumberValidator.attach(editText, context)
+
+        // Then
+        verify { editText.addTextChangedListener(any()) }
+    }
+
+    @Test
+    fun `validatePhoneNumber should set error message if phone number is invalid`() {
+        // Given
+        val editText = mockk<EditText>(relaxed = true)
+        val context = mockk<Context>()
+        every { context.getString(R.string.invalid_phone_message) } returns "Invalid phone number"
+        val invalidPhoneNumber = "+4812345678910"
+
+        // When
+        PhoneNumberValidator.validatePhoneNumber(invalidPhoneNumber, editText, context)
+
+        // Then
+        verify { editText.error = context.getString(R.string.invalid_phone_message) }
```

Comment:
verify { editText.error = "Invalid phone number" } should do the trick :)
Mockk has some issues comparing method calls like this, in most cases you need to pass an exact string so it can compare it properly